src/consumer/component-ops/add-components/add-components.js

```diff
@@ -478,17 +513,28 @@ export default class AddComponents {
 
   async add(): Promise<AddActionResults> {
     this.ignoreList = this.getIgnoreList();
+    this.ignoreList = this.ignoreList.map((ignorePattern) => {
+      if (ignorePattern.startsWith('**/')) {
+        return ignorePattern;
+      } else if (this.configuredConsumer === true) {
+        return path.join('**/', ignorePattern);
```

Comment:
This might cause unexpected ignored files. actually, I'm not sure we really need this at all...
We should check it.
And if we need a change to the ignore this is not the correct one.